src/commands/cmd_server.cc

```diff
@@ -978,6 +980,70 @@ static uint64_t GenerateConfigFlag(const std::vector<std::string> &args) {
   return 0;
 }
 
+class CommandRestore : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 4);
+    ttl_ms_ = GET_OR_RET(ParseInt<uint64_t>(args[2], 10));
+    while (parser.Good()) {
+      if (parser.EatEqICase("replace")) {
+        replace_ = true;
+      } else if (parser.EatEqICase("absttl")) {
+        auto now = util::GetTimeStampMS();
+        if (ttl_ms_ <= static_cast<uint64_t>(now)) {
+          return {Status::RedisExecErr, "Invalid expire time"};
+        }
+        ttl_ms_ -= now;
```

Comment:
in this case, redis will implicitly expire keys instead of throwing an error.
```
127.0.0.1:6379> restore busy_key 1111 "\x00\x01b\x0b\x00\x7f\xfb\x05\xe2[|\xfb\n" absttl
OK
```

and if we want to throw the error, here i think we should use RedisParseErr.

Reply:
Good catch, will fix it.